lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -165,8 +165,12 @@ SmallVector<unsigned> getContigPerThread(Attribute layout) {
     SmallVector<unsigned> contigPerThread(rank, 1);
     contigPerThread[rank - 1] = 2;
     return contigPerThread;
-  } else if (layout.isa<AMDMfmaEncodingAttr, AMDWmmaEncodingAttr>()) {
-    return {1, 1};
+  } else if (auto mfmaLayout = layout.dyn_cast<AMDMfmaEncodingAttr>()) {
```

Comment:
We should add case for AMDWmmaEncodingAttr. I'm currently not familiar with Wmma layout.